docs/index.md

```diff
@@ -5,49 +5,4 @@ feature_maturity: stable
 enterprise: 'no'
 ---
 
-Apache Spark is a fast and general-purpose cluster computing system for big data. It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. It also supports a rich set of higher-level tools including Spark SQL for SQL and DataFrames, MLlib for machine learning, GraphX for graph processing, and Spark Streaming for stream processing. For more information, see the [Apache Spark documentation][1].
```

Comment:
duplicate info, consolidated in parent: https://github.com/mesosphere/dcos-docs-enterprise/blob/4a9345aa626f7769df078a659d84a4f31c8628a5/service-docs/spark/index.md